testdata/invalidlineage/compat/change-default.txtar

```diff
@@ -29,6 +35,17 @@ lenses: [
             aunion: input.aunion
         }
     },
+    {
+        to: [0, 1]
+        from: [0, 2]
+        input: _
+        result: {
+            aunion: input.aunion
+        }
+    },
 ]
 -- out/bindfail --
-schema 0.1 must be backwards compatible with schema 0.0
+schema 0.2 is not backwards compatible with schema 0.1:
+field aunion not present in {aunion:*"foo" | "bar" | "baz"}:
+    /cue.mod/pkg/github.com/grafana/thema/lineage.cue:234:12
+missing field "aunion"
```

Comment:
This error _absolutely sucks_ - the `aunion` _field itself_ isn't missing, just some parts of its value!

But, it's still unambiguously better to be in a world where we have a true negative with a confusing error than a false positive with no error.